platform/fabric/core/generic/committer/committer.go

```diff
@@ -81,6 +82,8 @@ type Service struct {
 	FabricFinality     FabricFinality
 	Tracer             tracing.Tracer
 
+	logger committer.Logger
```

Comment:
can't we have a logging package under common/core instead?

Reply:
Sure let's align them all in a separate PR